### PR TITLE
create bimal.json

### PR DESCRIPTION
**Pull request by bimal-inc**
To make our job easier, please spend time to review your application before submitting. 

#### Add a x between [] if you meet that requirement example `[x]`
- [x] You're not using Vercel or Netlify.
- [x] You have completed your website, there's no type of placeholder at the website. **This requirement can be raised if the domain you're registering is for emails**
  - Link to website: 
- [x] The website is reachable.
- [x] The CNAME record doesn't contain any slash.
- [x] There's enough information at the `owner` field.
   - You should have your email presented. If you don't want to share email, you can leave email an empty string (`""`) and add any other social such as Discord/Twitter/etc.
 
 #### Feel free to join our discord server for more updates or any help related to the service :D - https://discord.gg/PZCGHz4RhQ

### CODE REVIEW

**Comment by bimal-inc** [comment]
Its been 2 days and my PR is still in pending...;(

**Comment by bimal-inc** [comment]
Its been 2 days and my PR is still in pending...;(

**Comment by MaskDuck** [comment]
dw i waited 1 month :)

File: domains/bimal.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "description": "My personal website goes here",
+  "repo": "https://github.com/bimal-inc/myportfolio",
+  "owner": {
+    "username": "bimal",
+    "email": "bimalyadavemail7@gmail.com",
+    "twitter": "@bimalinc"
+  },
+  "record": {
+    "CNAME": "bimal-inc.github.io/myportfolio/"
```
Comment (MaskDuck) [inline domains/bimal.json:10]: no slash in cname

**Review by MaskDuck — CHANGES_REQUESTED** [review]

File: domains/bimal.json (hunk 1 above)
Comment (bimal-inc) [reply domains/bimal.json]: Fixed now

**Comment by bimal-inc** [comment]
Fixed the CNAME , username and repo link @MaskDuck and thankyou for your  service

**Review by bimal-inc — COMMENTED** [review]
Fixed it should be working now

**Review by MaskDuck — APPROVED** [review]
my ancestor's feeling good

welcome https://bimal.is-a.dev !

(will be accesible in bimal.is-a.dev/myportfolio )

**Pull request merged**